book-service crud

decipherDev opened the pull request.

decipherDev committed:
book-service crud

spallya commented:
kindly follow the same comments which were given on the other PR of user service

decipherDev committed:
review comments

decipherDev committed:
code formatted

decipherDev commented:
@spallya I have addressed all the review comments. Swagger and flyway is in week two. I have completed it, I'll raise a separate PR once this is merged. 
Please review this PR in your free time.
